bitrise.yml

```diff
@@ -18,21 +20,23 @@ workflows:
     after_run:
     - test_xamarin
     - test_xcode
-    - test_tvos
+    - test_custom_plist
```

Comment:
the test workflow is missing from the CI

Reply:
What about this CI run? https://app.bitrise.io/build/a28de15ec6dada19#?tab=log

Reply:
Indeed. I think it would be good if all tests ran on Xcode 12 stack also.

It can be possibly changed in a way that the same workflow can be ran on multiple stacks by checking Xcode version https://github.com/bitrise-steplib/steps-xcode-test/blob/b103484d0955ecd24008835b907e6f30ee602cf9/bitrise.yml#L307.

Reply:
Running all tests on Xcode12